pyproject.toml

```diff
@@ -23,7 +23,7 @@ dependencies = [
     "requests>=2.32.3",
     "python-magic>=0.4.27",
     "httpx>=0.28.1",
-    "unicodedata2>=16.0.0",
+    #"unicodedata2>=16.0.0",
```

Comment:
I'm not sure why we need this package, but it fails to install when doing a Docker build. Do we really need it?